debug/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,6 @@
     <string name="debug_settings_test_device">Test Device</string>
     <string name="debug_settings_device_name">Device Name</string>
     <string name="debug_settings_device_id">Device ID</string>
+    <string name="debug_settings_set_device_id">Set Device ID</string>
+    <string name="debug_settings_device_id_description">Set the device ID to be used for testing</string>
```

Comment:
hey @Moux this is Device *Name* not ID. ID is something else

Reply:
Right.  Sorry, brain fart.  Fixed.